client/src/components/About.js

```diff
@@ -1,7 +1,27 @@
 import React from "react";
+import { Input, Menu, Header, Container, Grid } from "semantic-ui-react";
```

Comment:
I don't think these imports are all needed. Could you remove the extra ones pls?